spec/javascripts/saveSpec.js

```diff
@@ -255,4 +254,21 @@ describe('Saving a resource instance', function() {
     });
 
   });
+
+  describe('payload for save requests', function() {
+    var resource;
+    beforeEach(function() {
+      resource = Model.create({ name: 'foo', id: 12, subject: 'hello world!' });
+    });
+
+    it('saves requested fields when specified', function() {
+      resource.save({fields: ['name', 'id']});
```

Comment:
> Would we open source EditableResource to include it here?

I think we should rewrite `EditableResource` in EmberResouce. I think @shajith, @vcekov and I talked about this back when we were working on Agent Collision.

> Can you point me to an example situation where we want to save a model but we don't know what changed?

I think that's always the case, since instances are shared between consumers, there's a chance the same model instance could've been modified by a different consumer.